include/range/v3/range_fwd.hpp

```diff
@@ -393,12 +393,11 @@ namespace ranges
         struct ordered_less;
         struct ident;
 
-        enum cardinality
+        enum cardinality : std::ptrdiff_t
```

Comment:
AFAIK This is not an enum base (as in, inheriting cases from another enum) but just stating that the underlying storage of the enum will be of type `std::ptrdiff_t` (and hence `sizeof(cardinality) == sizeof(std::ptrdiff_t)`).